package-lock.json

```diff
@@ -13965,10 +13965,11 @@
       }
     },
     "node_modules/typescript": {
-      "version": "5.5.4",
-      "resolved": "https://registry.npmjs.org/typescript/-/typescript-5.5.4.tgz",
```

Comment:
If the latest version is 5.5.4, then why are we downgrading? Can't we update the in other places to use the latest typescript versions?

Reply:
Updated.